lib/hooks/useCopySnippet.ts

```diff
@@ -1,21 +1,23 @@
 import copy from 'copy-to-clipboard'
 import { useEffect } from 'react'
 
-const isCopyButton = (button) =>
+// TODO: TS support
+const isCopyButton = (button: any) =>
```

Comment:
HTMLButtonElement

Reply:
Line 24 `if (!(copyButton = isButtonChildren(e.target))) {`
it can't work

Reply:
Line 21 `if (isCopyButton(e.target)) {`
it can't work